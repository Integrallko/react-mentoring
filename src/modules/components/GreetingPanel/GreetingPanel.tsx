import React from 'react';

class GreetingPanel extends React.PureComponent {
	render() {
		return (	
			<div className='greet'>Hello World!</div>
		);
	}
}

export default GreetingPanel;
