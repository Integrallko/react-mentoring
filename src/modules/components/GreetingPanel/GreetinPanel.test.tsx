import React from 'react';
import renderer from 'react-test-renderer';

import GreetingPanel from './GreetingPanel';

const component = renderer.create(<GreetingPanel />,);
let greet = component.toJSON();

describe('GreetingPanel', () => {
    it('user should view component', () => {
        expect(greet).toMatchSnapshot();
    });
});
