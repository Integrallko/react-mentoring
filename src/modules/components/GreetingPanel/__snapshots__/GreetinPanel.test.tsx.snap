// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GreetingPanel user should view component 1`] = `
<div
  className="greet"
>
  Hello World!
</div>
`;
